Bottom: e36714b38d388624c6b5c5cf5e83385ba9c559e0
Top:    ed170bbc09384b43cc2771bc7894dd43cd7f7aca
Author: Jason Wang <jasowang@redhat.com>
Date:   2015-05-11 16:06:24 +0800

Refresh of vhost-interrupt-coalescing

---

diff --git a/drivers/vhost/vhost.h b/drivers/vhost/vhost.h
index faa0dd7..e0ec89c 100644
--- a/drivers/vhost/vhost.h
+++ b/drivers/vhost/vhost.h
@@ -166,6 +166,8 @@ void vhost_add_used_and_signal(struct vhost_dev *, struct vhost_virtqueue *,
 void vhost_add_used_and_signal_n(struct vhost_dev *, struct vhost_virtqueue *,
 			       struct vring_used_elem *heads, unsigned count);
 void vhost_signal(struct vhost_dev *, struct vhost_virtqueue *);
+void vhost_check_coalesce_and_signal(struct vhost_dev *dev,
+				     struct vhost_virtqueue *vq);
 void vhost_disable_notify(struct vhost_dev *, struct vhost_virtqueue *);
 bool vhost_enable_notify(struct vhost_dev *, struct vhost_virtqueue *);
